data_steward/analytics/table_metrics/unioned_ehr_scripts/unit_success_rate.py

```diff
@@ -383,18 +352,16 @@ def cstr(s, color='black'):
     `{DATASET}.concept` c
   ON
     m.unit_concept_id = c.concept_id
-LEFT JOIN
-`{DATASET}.concept_ancestor` ca
-ON
-ca.descendant_concept_id = m.measurement_concept_id
+  LEFT JOIN
+    `{DATASET}.union_concept_ancestor` ca
```

Comment:
Hi Nishanth, thanks for pointing this out. Yes, this union_ table was used because some concept_ids were found to be missing in the standard vocabulary table, due to vocab update. After Chao's work on concept_ids could be integrated to the standard table, this could be changed back. Let me know if you think otherwise

Reply:
I see, that makes sense!